### PR TITLE
Active events feature complete.

### PR DESCRIPTION
**Pull request by sunnybansal**
User can activate or deactivate an event.

### CODE REVIEW

File: ag_data/models.py [hunk 1]
```diff
@@ -53,6 +53,9 @@ class AGSensor(models.Model):
     name = models.CharField(max_length=1024, blank=True)
     type_id = models.ForeignKey(AGSensorType, null=False, on_delete=models.PROTECT)
 
+    # def __str__(self):
```
Comment (Ray310) [inline ag_data/models.py:56]: Why is this here and commented out?

**Comment by daisycrego** [comment]
@Ray310 lo quité
